script/release.sh

```diff
@@ -1,20 +1,20 @@
 #!/bin/bash
 TAG=${1-"latest"}
 
-make base
+make github
```

Comment:
maybe it can be changed to `make all`; "make all" builds images for every compiled nf.
The rest of the script can probably remain the same (maybe F5GC_BASE can be removed).
By this, no additional Dockerfile is needed and the script uses the same build procedure as users do to build their NFs.

Reply:
If the suggestion is ok, I think that 

```
cd base
git clone --recursive -b v3.2.1 -j `nproc` https://github.com/free5gc/free5gc.git
```
will need to be added at the beginning of the script.